manifests/setup/kustomization.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: kubesphere-logging-system
+commonLabels:
```

Comment:
Thanks for the PR @estahn 

The common label you add `app.kubernetes.io/name: fluentbit-operator` already exists in the following files:
- `fluentbit-operator-clusterRoleBinding.yaml` 
- `fluentbit-operator-deployment.yaml` 
- `fluentbit-operator-clusterRole.yaml`  
- `fluentbit-operator-serviceAccount.yaml`

And `fluentbit-operator-deployment.yaml` and `fluentbit-operator-serviceAccount.yaml` already have namespace `kubesphere-logging-system`

What you add here is a placeholder and maybe changed to other label or namespaces?

Maybe it's not necessary to add label `app.kubernetes.io/name: fluentbit-operator` to `namespace-namespace.yaml` and `fluentbit-operator-crd.yaml`

Reply:
Good points. The adjustments should be made downstream. I also removed the namespace resource as this is something that should done downstream too.